cherrypy/lib/httputil.py

```diff
@@ -501,18 +512,21 @@ class Host(object):
     name = 'unknown.tld'
 
     def __init__(self, ip, port, name=None):
+        """Initialize Host."""
         self.ip = ip
         self.port = port
         if name is None:
             name = ip
         self.name = name
 
     def __repr__(self):
+        """Represent Host object."""
         return 'httputil.Host(%r, %r, %r)' % (self.ip, self.port, self.name)
 
 
 class SanitizedHost(str):
-    r"""
+    r"""Sanitize host header.
```

Comment:
This isn't a function, so it shouldn't start with a verb:
```suggestion
    r"""A normalized host header value.
```